draft-ietf-oauth-v2-1.md

```diff
@@ -2265,6 +2267,18 @@ server that mistakenly believes the token to be for it.
 An attacker attempts to use an access token that has already
 been used with that resource server in the past.
 
+#### Intermediaries
+
+Since  HTTP infrastructures rely significantly on intermediaries
+(Section 3.7 of {{SEMANTICS}}),
+caches and TLS terminators (e.g. load balancers, API gateways)
+those actors might have visibility of access tokens.
```

Comment:
is it visibility only? I would assume forgery is a risk, too, that's why implementers must sign tokens or use high entropy handles